modules/Vtiger/models/Record.php

```diff
@@ -486,7 +486,9 @@ public static function getCleanInstance($moduleName)
 		$instance = new $modelClassName();
 		$instance->setModuleFromInstance($module);
 		$instance->isNew = true;
-		$instance->setData($focus->column_fields)->setModule($moduleName)->setEntity($focus);
+		if (property_exists($focus, 'column_fields')) {
+			$instance->setData($focus->column_fields)->setModule($moduleName)->setEntity($focus);
```

Comment:
this one is caused by cron OSSMailScanner and OSSMail check

~~~
PHP Notice:  Undefined property: OSSMailScanner::$column_fields in modules/Vtiger/models/Record.php on line .... 
PHP Stack trace: 
PHP   1. {main}() cron/vtigercron.php:0 
PHP   2. require_once() cron/vtigercron.php:74 
PHP   3. Vtiger_Record_Model::getCleanInstance() modules/OSSMailScanner/cron/action.php:12 
~~~

 Both are not extending CRMEntity.